assets/stylesheets/base/_padding.scss

```diff
@@ -0,0 +1,20 @@
+
+.content-wrap {
```

Comment:
Based on changing the above file, this one seems more appropriate as `_layout.scss`.